examples/banking/account/account.go

```diff
@@ -114,3 +100,19 @@ func (aggregateRoot) HandleMessage(
 
 	return
 }
+
+// ApplyEvent updates the data to reflect the fact that ev has occurred.
+func (aggregateRoot) ApplyEvent(d saga.Data, env ax.Envelope) {
```

Comment:
Note that this method is now on the `aggregateRoot` (the `saga.Saga` implementation), not the saga data itself.